rcp/org.testeditor.rcp4.views.tcltestrun/.classpath

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
+	<classpathentry exported="true" kind="lib" path="lib/javax.json-1.0.4.jar"/>
```

Comment:
we should not add external libs if not absolutely required - which is not the case here